static/src/stylesheets/module/identity/_wrapper.garnett.scss

```diff
@@ -0,0 +1,77 @@
+//TODO: most of this is not actually needed post-switch it's just overrides
```

Comment:
I would make this fairly explicit maybe:
`Post garnett launch (January 15th, 2018) most of this wont be needed/will need to be refactored as it is all overrides`